master/pkg/tasks/task_generic.go

```diff
@@ -99,7 +99,7 @@ func (s GenericTaskSpec) SetJobPriority(priority int) error { return nil }
 
 // SetWeight todo.
 func (s GenericTaskSpec) SetWeight(weight float64) error {
-	s.Base.ResourcesConfig.SetWeight(weight)
+	s.GenericTaskConfig.Resources.SetWeight(weight)
```

Comment:
will this actually affect priority in scheduling?

I see commands set the group weight in the resource manager

https://github.com/determined-ai/determined/blob/ba03375270d0c63620c4171a35a40d89ff4bf89a/master/internal/command/command_job_service.go#L62-L72

Reply:
The implementation of `toV1Job` and `SetWeight` was originally to just allow `det job list` to work when a generic task was running. We will still need to fully implement all the functions later. If you see the priority and resource pool functions are still unimplemented